cocos2d/core/components/CCLayout.js

```diff
@@ -408,7 +408,7 @@ var Layout = cc.Class({
         var row = 0;
         var containerResizeBoundary;
 
-        var maxHeightChildAnchor;
```

Comment:
Didn't see anywhere to use `maxHeightChildAnchor.x` and it's not a good practice to create new Point object here, we don't need to, if only y is needed, please use `maxHeightChildAnchorY` otherwise, please use `maxHeightChildAnchorX` and `maxHeightChildAnchorY`